extensions/amp-video/0.1/amp-video.js

```diff
@@ -765,7 +765,18 @@ export class AmpVideo extends AMP.BaseElement {
    */
   setUpCaptions_() {
     const captionsId = this.element.getAttribute('captions-id');
-    if (!captionsId) {
+    // if (!captionsId) {
+    //   return;
+    // }
+
+    const trackEl = this.element.querySelector('track');
+    if (!captionsId && trackEl) {
+      // Works because this function is called twice, so this function runs again after it is attached from the first time it's called.
+      const captionsId = trackEl.getAttribute('src');
+      const captionsEl = this.win.document.createElement('amp-story-captions');
+      captionsEl.setAttribute('id', captionsId);
```

Comment:
We would probably want to use the `default` preset when this is not present right? I wonder however, what benefits come from using custom captions rather than default ones when the story doesn't explicitly declare this.